### ISSUE TITLE
phasic content fusing model

### ISSUE THREAD

**Issue by fikry102** [sjtuplayer/few-shot-diffusion#4]
`After the basic diffusion model is obtained, you can train phasic content fusing module by:
python3 train-recon.py --data_path=$path_to_dataset --ckpt=$path_to_basic_diffusion_checkpoint
`
So, we only use source dataset to train the phasic content fusing module?
How should it be related to Figure 2 in the paper?
![image](https://github.com/sjtuplayer/few-shot-diffusion/assets/63465970/9a36cdbc-43b9-4772-bbd9-24e927cbf995)

**Comment by sjtuplayer** [comment]
Yes, we only use the source-domain dataset to train the phasic content fusing module in step 2 since this step is only used for initializing the module. Figure 2 is the framework of `train-whole.py`, which trains the diffusion model and the phasic content fusing module at the same time.

**Comment by invade-art** [comment]
> Yes, we only use the source-domain dataset to train the phasic content fusing module in step 2 since this step is only used for initializing the module. Figure 2 is the framework of `train-whole.py`, which trains the diffusion model and the phasic content fusing module at the same time.

so? why train the phasic content fusing module in step2

**Comment by sjtuplayer** [comment]
> > Yes, we only use the source-domain dataset to train the phasic content fusing module in step 2 since this step is only used for initializing the module. Figure 2 is the framework of `train-whole.py`, which trains the diffusion model and the phasic content fusing module at the same time.
> 
> so? why train the phasic content fusing module in step2

the PCF module is used for keeping the identity consistency during domain adaption.
